internal/slicer/slicer.go

```diff
@@ -397,3 +391,63 @@ func createFile(targetPath string, pathInfo setup.PathInfo) (*fsutil.Entry, erro
 		MakeParents: true,
 	})
 }
+
+// selectPkgArchives selects the appropriate archive for each selected slice
+// package. It returns a map of archives indexed by package names.
+func selectPkgArchives(archives map[string]archive.Archive, selection *setup.Selection) (map[string]archive.Archive, error) {
+	// Sort the archives by their priority value.
+	releaseArchives := make([]*setup.Archive, 0, len(selection.Release.Archives))
```

Comment:
Can you change the name? These are not the archives from the release anymore now that we are ignoring the negative priorities. Thoughts about `sortedArchives` or `archivesSorted` ?

Reply:
I like `sortedArchives`, gracias.